tests/version_test.py

```diff
@@ -34,10 +34,8 @@ def test_version_is_semver():
 def test_input_file_version(tmp_path):
```

Comment:
**suggestion (non-blocking):** could now use `semver.Version.is_valid()` above in `test_version_is_semver()`